File: clients/go/my_strategy.go
package main

import (
    . "spb_ai_champ/model"
)

type MyStrategy struct{}

func NewMyStrategy() MyStrategy {
    return MyStrategy{}
}

func (strategy MyStrategy) getAction(game Game) Action {
    return Action{
        Moves: make([]MoveAction, 0),
        Buildings: make([]BuildingAction, 0),
    }
}

File: clients/rust/src/my_strategy.rs
use spb_ai_champ::model::*;

pub struct MyStrategy {}

impl MyStrategy {
    pub fn new() -> Self {
        Self {}
    }
    pub fn get_action(&mut self, game: &Game) -> Action {
        Action {
            moves: vec![],
            buildings: vec![],
            choose_specialty: None,
        }
    }
}
